lib/interface_quda.cpp

```diff
@@ -4621,6 +4647,7 @@ void computeHISQForceQuda(void* const milc_momentum,
 
   profileHISQForce.TPSTART(QUDA_PROFILE_COMPUTE);
   hisqStaplesForce(*cudaOutForce, *cudaInForce, *cudaGauge, act_path_coeff);
+  qudaDeviceSynchronize();
```

Comment:
I guess all those qudaDeviceSynchronize() are maing sure we get correct timing but do they add overhead?

Reply:
That was a @maddyscientist addition so I defer to her, I didn't explicitly check that contribution, but any cost certainly gets amortized by the e2e speed-up.

Reply:
Discussed offline, cost is negligible (especially relative to 100s of force kernels inside the function)